src/llvm-coverage-instrumentation.md

```diff
@@ -496,7 +496,7 @@ An visual, interactive representation of the final `CoverageSpan`s can be
 generated with the following `rustc` flags:
 
 ```shell
-$ rustc -Zinstrument-coverage -Zdump-mir=InstrumentCoverage \
+$ rustc -Z winstrument-coverage -Zdump-mir=InstrumentCoverage \
```

Comment:
typo:


```suggestion
$ rustc -Z instrument-coverage -Zdump-mir=InstrumentCoverage \
```

Reply:
Good catch, thanks! This PR is still in draft because I can't actually run the commands until I'm at my desktop.

Reply:
All the commands looked fine to me!

By the way, while you're at it you might want to add a space to make these consistent:

```suggestion
$ rustc -Z instrument-coverage -Z dump-mir=InstrumentCoverage \
```

Reply:
Grepped the entire book for `'-Z[^\s]'`and `'-C[^\s]'` and standardized on a following space.